File: tundra-cli/benchmarks/iterative-fibonacci/fib.cs
using System;
class Fib01 {
  static void Main() {
    int n = 80;
    long a = 0, b = 1;
    for (int i=0; i<n; i++) {
      var tmp = a + b;
      a = b;
      b = tmp;
    }
    Console.WriteLine(a);
  }
}

File: tundra-cli/benchmarks/iterative-fibonacci/fib.cpp
#include <iostream>
int main() {
    int n = 80;
    long a = 0, b = 1;
    for (int i = 0; i < n; i++) {
        auto tmp = a + b;
        a = b;
        b = tmp;
    }
    std::cout << a << std::endl;
    return 0;
}